core/templates/dev/head/domain/utilities/BrowserCheckerService.js

```diff
@@ -26,9 +26,9 @@ oppia.factory('BrowserCheckerService', [
       function(p) {
         return p.toString() === '[object SafariRemoteNotification]';
       })(
-        !window.safari ||
+      !window.safari ||
```

Comment:
Since this being passed as args and follows from the previous line, the spacing before should be preserved.

Reply:
Eslint produces an error if we try to retain the spacing here. This might be because the arguments are already indented by 2 from `var isSafari...`.

Reply:
Ah, okay.